Common/UnitDefinitions/Humitidy.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Name": "Humidity",
```

Comment:
I'm not so sure about adding this quantity, it seems to overlap a lot with `Ratio` percentage. Is that an option for you to use? You would lose the `%Rh` abbreviation out of the box though.

We generally want to avoid adding quantities that are not convertible - meaning they only have a single unit (no units to convert between) and there are no other quantities to convert to (like `Speed = Length / Duration`). We are simply trying to avoid bloat, because the number of measurable quantities is very large.

If you can think of additional units or quantities to convert to, that would help justify adding humidity as its own quantity type.